src_c/base.c

```diff
@@ -1578,7 +1588,7 @@ _pg_values_as_buffer(Py_buffer *view_p, int flags, PyObject *typestr,
                         "'shape' and 'strides' are not the same length");
         return -1;
     }
-    view_p->ndim = ndim;
+    view_p->ndim = (int)ndim;
```

Comment:
This is surprising that ndim is an int. I guess it's not possible to create such large dimensions.

Reply:
yeah, true. `ndim` would usually be from 1 to 3 for most practical purposes, right?